crates/nu-command/src/database/values/sqlite.rs

```diff
@@ -20,16 +24,24 @@ pub struct SQLiteDatabase {
     // 1) YAGNI, 2) it's not obvious how cloning a connection could work, 3) state
     // management gets tricky quick. Revisit this approach if we find a compelling use case.
     pub path: PathBuf,
+    #[serde(skip)]
+    // this understandably can't be serialized. think that's OK as long as we
```

Comment:
don't leave me hanging bro. "think that's OK as long as we"... what? 🤣

Reply:
oops! good catch, was gonna say it's OK because this isn't being used in a plugin